load/UpdateDBPAccessTable.py

```diff
@@ -83,7 +88,7 @@ def process(self, filesetList):
 
 	def _isSILOnly(self, record):
```

Comment:
isSILOnly can be deleted since "SIL" is added to the org list

Reply:
@bradflood It has been removed and I have also removed the method `_isPioneerBibleTranslatorsOnly`. See: https://github.com/faithcomesbyhearing/dbp-etl/pull/117/files#diff-1fd781079e6b9caf6a8d8693ceb7aeb9b93a55e172acd2406273d04f4c1a5c1bL84